pkg/volume/fc/fc_util.go

```diff
@@ -101,15 +140,28 @@ func scsiHostRescan(io ioHandler) {
 	}
 }
 
+// check scsi disk is really exist and is attached to the node using SCSI Inquiry
+func checkDiskExist(b fcDiskMounter, disk, wwid string) (bool, error) {
+	out, err := b.plugin.execCommand("/lib/udev/scsi_id", []string{"--replace-whitespace", "--page=0x83", "--whitelisted", "--device=" + disk})
```

Comment:
I can find `/lib/udev/scsi_id` on Fedora based containers, but not on latest ubuntu containers.